source/store/CaseContext.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { post } from 'app/helpers/ApiRequest';
+import AuthContext from 'app/store/AuthContext';
+import casesMock from '../assets/mock/cases';
+
+const CaseContext = React.createContext();
+
+export const CaseConsumer = CaseContext.Consumer;
+
+export function CaseProvider({ children }) {
+  const { user } = useContext(AuthContext);
+  const [cases, setCases] = useState([]);
+  const [fetching, setFetcing] = useState(false);
+
+  useEffect(() => {
+    // Todo Replace with api request towards AWS.
+    setFetcing(true);
+
+    setTimeout(() => {
+      setCases(casesMock);
+      setFetcing(false);
+    }, 200);
+  }, [cases]);
```

Comment:
Looks Strange when the case is set here as well as useEffect will render depending on cases.  May be it should depend on the user

Reply:
Agree